xbmc/windowing/WinSystem.h

```diff
@@ -181,6 +182,32 @@ class CWinSystemBase
 
   virtual std::vector<std::string> GetConnectedOutputs() { return {}; }
 
+  /*!
+   * @brief Queries the age of the current backbuffer.
+   *
+   * @return Returns the buffer age. Usual values are: 1 for double buffering,
+   * 2 for triple buffering.
+   *
+   * @note Should be called once before any draw call, as it triggers damaged
+   * regions rendering.
+   */
+  virtual int32_t GetBufferAge() { return 2; }
```

Comment:
Why int32_t? Why not just int or unsigned int?

Reply:
Not particular reason. The query for GL returns an EGLInt, which is an int32_t as far as I know. Plus I had the idea to pass -1 for an unknown age.

What do you want me change it to?

Reply:
Using a sized int is making an assertion on the maximum value. Here, you're saying that the buffer age is < 2^31, but if it can only be 1 or 2, the assertion doesn't make sense, so I would use `unsigned int` to show that the bitsize isn't very important to the meaning.